modules/cluster_estimation/cluster_estimation.py

```diff
@@ -23,16 +23,16 @@ class ClusterEstimation:
     ATTRIBUTES
     ----------
     min_activation_threshold: int
-        Minimum total data points before model runs.
+        Minimum total data points before model runs. Must be at least 1.
 
     min_new_points_to_run: int
-        Minimum number of new data points that must be collected before running model.
+        Minimum number of new data points that must be collected before running model. Must be at least 0.
 
     max_num_components: int
-        Max number of real landing pads.
+        Max number of real landing pads. Must be greater than 0.
 
     random_state: int
-        Seed for randomizer, to get consistent results.
+        Seed for randomizer, to get consistent results. Must be at least 0.
```

Comment:
Remove attributes header and move these descriptions into the `create()` docstring.